pages/en/developer/create-subgraph-hosted.mdx

```diff
@@ -24,7 +24,7 @@ The Graph Network supports subgraphs indexing mainnet Ethereum:
 - `xdai`
 - `near-mainnet`
 - `near-testnet`
-- `matic`
```

Comment:
This naming is currently fixed in the Hosted Service. There is an [open issue](https://github.com/graphprotocol/graph-node/issues/3075) that will enable easy switches such as this, but we can't change this naming at this time (we can perhaps add a note to indicate that this is "Polygon" chain)